jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -476,5 +476,10 @@ def set_plot_axes(self):
         # Make it so y axis label is not covering tick numbers.
         self.figure.axes[1].label_offset = "-50"
 
-        # # Set Y-axis to scientific notation
+        # Set Y-axis to scientific notation
         self.figure.axes[1].tick_format = '0.1e'
+
+        # Change collapse function to sum and
+        # reset axes limits to match new spectrum view
+        self.state.function = 'sum'
+        self.state.reset_limits()
```

Comment:
the `reset_limits` won't be needed with https://github.com/glue-viz/glue/pull/2277